packages/babel-code-frame/src/index.js

```diff
@@ -127,8 +127,8 @@ function getMarkerLines(
     loc.start,
   );
   const endLoc: Location = Object.assign({}, startLoc, loc.end);
-  const linesAbove = opts.linesAbove || 2;
-  const linesBelow = opts.linesBelow || 3;
+  const linesAbove = isNaN(opts.linesAbove) ? 2 : opts.linesAbove;
+  const linesBelow = isNaN(opts.linesBelow) ? 3 : opts.linesBelow;
```

Comment:
`isNaN` is imho a legacy API and should not be used nowadays, not sure what the intention was but I guess this should be fine, right?
```
const linesAbove = typeof opts.linesAbove !== 'number' ? 2 : opts.linesAbove;
const linesBelow = typeof opts.linesBelow !== 'number' ? 3 : opts.linesBelow;
```

Reply:
seems like it's just to allow 0

Reply:
yes, to allow 0. I agree about the isNaN usage - i would have preferred to use destructuring with defaults. But, guess I was going for lowest impacting change lol.